containers/ecr-viewer/src/app/layout.tsx

```diff
@@ -28,14 +31,18 @@ export default function RootLayout({
       <html lang="en">
         <head />
         <body className="overflow-x-auto">
-          <div className="position-relative isolate">
-            <AuthSessionProvider>
+          <AuthSessionProvider>
+            <Header>
+              <NavLinks />
+            </Header>
+            <div className="position-relative isolate">
               <ToastProvider>
                 {children}
                 <ToastShelf />
               </ToastProvider>
-            </AuthSessionProvider>
-          </div>
+              <Footer />
+            </div>
+          </AuthSessionProvider>
```

Comment:
```suggestion
          </AuthSessionProvider>
          </div>
```

We still want this div around everything to isolate the app from the modals